client/views/teams/contextualBar/info/Delete/index.js

```diff
@@ -1,20 +1,25 @@
 import React, { useMemo } from 'react';
 
+import { AsyncStatePhase } from '../../../../../hooks/useAsyncState';
 import { useEndpointData } from '../../../../../hooks/useEndpointData';
 import DeleteTeamModal from './DeleteTeamModal';
+import DeleteTeamSkeleton from './DeleteTeamSkeleton';
 import StepOne from './StepOne';
-import StepThree from './StepThree';
 import StepTwo from './StepTwo';
 
 const DeleteTeamModalWithRooms = ({ teamId, onConfirm, onCancel }) => {
-	const { value } = useEndpointData(
+	const { value, phase } = useEndpointData(
 		'teams.listRooms',
 		useMemo(() => ({ teamId }), [teamId]),
 	);
 
+	if (phase === AsyncStatePhase.LOADING) {
+		return <DeleteTeamSkeleton onClose={onCancel} />;
```

Comment:
I don't think it's necessary to create a separate component for the Skeleton state since we have the generic one, we could use it directly